File: Retrofit Upload File/You_Part/app/src/main/java/com/example/you_part/ResponseModel.java
package com.example.you_part;


import com.google.gson.annotations.SerializedName;
import java.io.Serializable;

public class ResponseModel implements Serializable {

	@SerializedName("data")
	private Data data;

	@SerializedName("success")
	private boolean success;

	@SerializedName("status")
	private int status;

	public Data getData(){
		return data;
	}

	public boolean isSuccess(){
		return success;
	}

	public int getStatus(){
		return status;
	}
}

File: Unit5 Evaluation/Evaluation-2 NewsBreeze App/app/src/main/java/com/application/newsbreezeapp/ApiCall/ResponseModel.kt
package com.application.newsbreezeapp.ApiCall


import com.google.gson.annotations.SerializedName

data class ResponseModel(
    @SerializedName("articles")
    val articles: List<Article>,
    @SerializedName("status")
    val status: String,
    @SerializedName("totalResults")
    val totalResults: Int
)